src/applications/mhv/landing-page/mocks/api/index.js

```diff
@@ -6,13 +6,14 @@ const commonResponses = require('../../../../../platform/testing/local-dev-mock-
 const featureToggles = require('./feature-toggles/index');
 const user = require('./user/index');
 const folders = require('./mhv-api/messaging/folders/index');
+const vamcEhr = require('../../tests/fixtures/vamc-ehr.json');
 
 const responses = {
   ...commonResponses,
   'GET /v0/user': user.defaultUser,
   'GET /v0/feature_toggles': featureToggles.generateFeatureToggles({}),
   'GET /my_health/v1/messaging/folders': folders.allFoldersWithUnreadMessages,
-  'GET /data/cms/vamc-ehr.json': {},
+  'GET /data/cms/vamc-ehr.json': vamcEhr,
```

Comment:
Since these are API mocks, this gets served as `http://localhost:3001/data/cms/vamc-ehr.json` on port `3001`, whereas the requests I see when loading the landing page are to port `3000`… 

I don't think this gets used

Reply:
Fwiw, I had to `cp ../content-build/build/localhost/data/cms/vamc-ehr.json build/localhost/data/cms/` to get that file served locally.

Reply:
This would get used when developing with the `mock-api`. I suppose now that we are moving away from needing to check details about a user's associated facilities, this won't be necessary.